packages/app/src/game/module/board/overlays/readyOverlay.tsx

```diff
@@ -0,0 +1,62 @@
+import { getUserId } from "packages/app/src/auth";
+import { PlayerListPlayer, StreakType } from "packages/models/lib";
+import * as React from "react";
+import { useSelector } from "react-redux";
+import { AppState } from "../../../../store";
+import { BoardOverlay } from "./boardOverlay";
+
+const getStreakType = (player: PlayerListPlayer): string => {
+
+    const streakType = player?.streakAmount === 0 ? ""
+        : player?.streakType === StreakType.WIN ? player?.streakAmount === 1 ? "Win" : "Wins"
+            : player?.streakAmount === 1 ? "Loss" : "Losses"
+
+    return streakType
+}
+
+const getPosition = (player: PlayerListPlayer): number => {
+    const position = useSelector((state: AppState) => {
```

Comment:
`useSelector` can only be used in component render functions. To get around this you could make `getPosition` take `(player: PlayerListPlayer, playerList: PlayerListPlayer[])`, and then put the `useSelector` in the component, passing both in here

Reply:
* Moved useSelector as suggested and passed playerList to getPosition function